script.js

```diff
@@ -1,4 +1,1250 @@
+var Code01 = "PROJECT PART 1: SCISSORS PAPER STONE: PART 1-STANDARD MODE";
+
+// random number generator
+
+/* var randomNumber = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  return randomInteger + 1;
+};
+
+// read random Number to output  random GameHand
+var readGameHandNmbr = function (gameHandNmbr) {
+  var gameHandPlayed = "";
+  if (gameHandNmbr == 1) {
+    gameHandPlayed = "scissors";
+  }
+  if (gameHandNmbr == 2) {
+    gameHandPlayed = "paper";
+  }
+  if (gameHandNmbr == 3) {
+    gameHandPlayed = "stone";
+  }
+  return gameHandPlayed;
+};
+
+var main = function (input) {
+  var gameHandRandom = randomNumber();
+  var gameHand = readGameHandNmbr(gameHandRandom);
```

Comment:
While having helper functions is rarely a bad idea, I would say it would be good to put the readGameHandNmbr together with the randomNumber function to just generate a function called compChoice for example - it would be less confusing unless you need to reuse the randomNumber function again (which is not the case for SPS)